source/_patterns/01-molecules/item-details/item-details.twig

```diff
@@ -0,0 +1,49 @@
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['item-details']) }) -%}
+
+<div {%- include('atoms-attributes') -%}>
+
+  {%- with list only -%}
+    {%- block list -%}
+
+      {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['item-detail-tag']) }) -%}
+
+      {%- if itemDetails|default(false) -%}
+        {%- set attributes = attributes|merge({class: attributes.class|merge(['item-detail-tag__item']) }) -%}
```

Comment:
This is weird. `itemDetails` variable doesn't exist; not sure what `item-detail-tag` is.

Reply:
I should have taken this out. This has been done